src/test/resources/pages/instructorSearchPageSearchComments.html

```diff
@@ -135,7 +135,7 @@
                   </b>
                   [Tue, 01 May 2012, 08:49 AM UTC]
                 </span>
-                <a class="btn btn-default btn-xs icon-button pull-right" data-original-title="Edit comment in the Comments page" data-placement="top" data-toggle="tooltip" href="/page/instructorCommentsPage?user=searchUI.idOfInstructor1OfCourse1&courseid=searchUI.idOfTypicalCourse1#${comment.id}" style="display:none;" target="_blank" title="" type="button">
+                <a class="btn btn-default btn-xs icon-button pull-right" data-original-title="Edit comment in the Comments page" data-placement="top" data-toggle="tooltip" href="/page/instructorCommentsPage?user=searchUI.idOfInstructor1OfCourse1&courseid=searchUI.idOfTypicalCourse1#${comment.id}" style="" target="_blank" title="" type="button">
```

Comment:
It's strange. What about FF version, as I remember it was updated for testing. I am still on previous one. May it be the reason ?